chonchon/metallb/metallb.sh

```diff
@@ -1,10 +1,9 @@
 #!/bin/bash
 
 set -ex
-
-kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.12.1/manifests/namespace.yaml
-kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.12.1/manifests/metallb.yaml
-# On first install only
-kubectl create secret generic -n metallb-system memberlist --from-literal=secretkey="$(openssl rand -base64 128)"
-
-kubectl apply -f ./mlb-l2.yaml
+helm repo add metallb https://metallb.github.io/metallb
+helm --namespace metallb-system \
+    install --create-namespace \
```

Comment:
Please use `update --install` instead of `install` so that the script can be rerun without modification.

Reply:
thats a good idea, i will make the changes to the scripts. About the pools, i just keep the same pools that were configure before, we will change the yagan one for the requierement later on. Thx